Index: Steganography.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from libs import audio_steganography\r\nfrom libs import image_steganography\r\nfrom libs import text_steganography\r\nfrom libs import vedio_steganography\r\n\r\n\r\ndef main():\r\n    pass\r\n\r\n\r\nif __name__ == \"__main__\":\r\n    main()\r\n\r\n# MAIN MENU\r\n#\r\n# 1. IMAGE STEGANOGRAPHY {Hiding Text in Image cover file}\r\n# 2. TEXT STEGANOGRAPHY {Hiding Text in Text cover file}\r\n# 3. AUDIO STEGANOGRAPHY {Hiding Text in Audio cover file}\r\n# 4. VIDEO STEGANOGRAPHY {Hiding Text in Video cover file}\r\n# 5. Exit\r\n\r\n\r\n# SYS\r\n# python Steganography.py -<e-->encode> <file location>\r\n# python Steganography.py -<d-->decode> <file location>\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Steganography.py b/Steganography.py
--- a/Steganography.py	(revision 5f262827bf097849b73d954e7a92b396f521ad48)
+++ b/Steganography.py	(date 1669273184660)
@@ -20,7 +20,7 @@
 # 5. Exit
 
 
-# SYS
+# SYS![](../../AppData/Local/Temp/download.jpg)
 # python Steganography.py -<e-->encode> <file location>
 # python Steganography.py -<d-->decode> <file location>
 
